### ISSUE TITLE
Any open issues for a 0.9.0 release?

### ISSUE THREAD

**Issue by cwe1ss** [opentracing/opentracing-csharp#27]
Besides the two open PRs #13 and #26, I'm pretty happy with the current code. If we can resolve the PRs until then, I'd like to release a 0.9.0 next week. (the current nuget.org package has the version 0.8.2)

Do you have any other open issues that need to be addressed?

/cc @bensigelman @dawallin @yurishkuro

**Comment by cwe1ss** [comment]
#28 (restricted tag types) should go in as well.

**Comment by dawallin** [comment]
LGTM, I have nothing more to add.

**Comment by cwe1ss** [comment]
I made a final review in 7a2b564 and changed the following things:

* I reviewed the docs and moved them closer to the Java docs.
* I changed `Finish(DateTime? finishTimestamp = null)` to two separate methods (`Finish()` and `Finish(DateTime finishTimestamp)`. This is exactly the same for users but it allowed me to write separate docs for each case. 
* I added an additional `SetTag`/`WithTag` overload for `int` because I read that converting int to double isn't optimal. Now we should definitely have the most common cases.
* The library now targets .NET standard 1.0 instead of 1.1, which means it supports really every .NET platform that's out there. Seems like 1.1 was a relic from some time ago where we needed newer APIs.

I didn't do a PR for this because it doesn't change anything notable IMO. I hope that's ok for everyone.

@bensigelman @dawallin @yurishkuro Any final comments from your side? If not, I'll probably release it on Friday!
